doc-content/kogito-docs/src/main/asciidoc/creating-running/ref_kogito-app-examples.adoc

```diff
@@ -0,0 +1,14 @@
+[id='ref_kogito-app-examples']
+
+= {PRODUCT} example applications
```

Comment:
good, I am going to merge this repository with the quickstarts repository soon. there will be more examples, and some these will be gone. I'll send you updates when this happens